bin/rpc.ml

```diff
@@ -91,8 +91,9 @@ module Status = struct
     let+ (common : Common.t) = Common.term in
     client_term common @@ fun _common ->
     let where = active_server () in
-    printfn "Server is listening on %s" (Dune_rpc.Where.to_string where);
-    printfn "Connected clients (including this one):\n";
+    Dune_console.printf "Server is listening on %s"
```

Comment:
Just use `Console.print`. You can also print both messages in a single call.

Reply:
Good point.